dotcom-rendering/src/web/components/ShowMore.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import fetchMock from 'fetch-mock';
+import { trails } from '../../../fixtures/manual/show-more-trails';
+import { ShowMore } from './ShowMore.importable';
+
+const containerTitle = 'Opinion';
+const path = 'uk/lifestyle';
+const containerId = '5011-3940-8793-33a9';
+
+export default {
+	component: ShowMore,
+	title: 'Components/ShowMore',
+};
+
+export const ShowMoreSuccess = () => {
```

Comment:
Currently these stories will only show the success/error states if a user clicks on them. So they're useful for documentation purposes but not so useful for Chromatic.

I don't know if we have a good way of showing the component in an 'open' state in Storybook, or whether we should wait until we've decided how we want to use Percy?

Reply:
Alternatively you could pass the initial isOpen state as a prop and default it to `false`